src/makers/makeStates.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * @module makeStates
+ *
+ * @description
+ *
+ * Create SMACSS style state rules.
+ * If the correspondent props value is truthy,
+ * class with `is-` prefix will be added.
+ *
+ * Supports `active`, `disabled`, `hidden`, and `loading`.
```

Comment:
http://cdn1.theodysseyonline.com/files/2015/10/24/635812585725464155-2069391564_Strippers.jpg
On Fri, 15 Apr 2016 at 15:52, Alexander Gudulin <notifications@github.com>
wrote:

> In src/makers/makeStates.js
> <https://github.com/agudulin/dumb-bem/pull/9#discussion_r59876766>:
>
> > @@ -0,0 +1,29 @@
> > +/**
> > + * @module makeStates
> > + *
> > + * @description
> > + *
> > + * Create SMACSS style state rules.
> > + * If the correspondent props value is truthy,
> > + * class with `is-` prefix will be added.
> > + *
> > + * Supports `active`, `disabled`, `hidden`, and `loading`.
>
> not sure about this comma
>
> - Supports `active`, `disabled`, `hidden`, and `loading`.+ Supports `active`, `disabled`, `hidden` and `loading`.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/agudulin/dumb-bem/pull/9/files/0b4228fe11925a9e283cf089ed41598abe6fb50c#r59876766>
>